packages/main/src/components/protected/topbar/components/Profile.jsx

```diff
@@ -201,6 +201,24 @@ const Profile = () => {
           <div className={styles.infoTitle}>Local time</div>
           <div className={styles.infoContent}>{currentTime}</div>
         </div>
+
+        {/**<div className={styles.social}>
+          <div className={styles.icons}>
+            <img src={linkedin} alt="linkedin" />
+          </div>
+          <div className={styles.icons}>
+            <img src={facebook} alt="linkedin" />
+          </div>
+          <div className={styles.icons}>
+            <img src={github} alt="linkedin" />
+          </div>
+          <div className={styles.icons}>
+            <img src={twitter} alt="linkedin" />
+          </div>
+          <div className={styles.icons}>
+            <img src={instagram} alt="linkedin" />
+          </div>
+            </div> */}
```

Comment:
Remove this commented out code if its not being used

Reply:
On it right now

Reply:
I have removed them